app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,27 @@
+class SessionsController < ApplicationController
+  def create
+    user = User.find_by_email(params[:email])
+
+    if user && user.valid_password?(params[:password])
+      @current_user = user
```

Comment:
why do we need instance variable? can't we just use the variable `user`?

Reply:
Thank you for the comment. I removed the instance variable!